.github/workflows/build.yml

```diff
@@ -205,15 +232,15 @@ jobs:
           submodules: recursive
           fetch-depth: 2 # default is 1 and codecov needs > 1
 
-      # We use macOS for the final publishing build so we get all the iOS/macCatalyst targets in the packages
+      # We use macOS for the final publishing build so we get all the iOS/Mac Catalyst targets in the packages
       - name: Set Environment Variables
         run: echo "CI_PUBLISHING_BUILD=true" >> $GITHUB_ENV
 
       - name: Download sentry-native (macOS)
         uses: actions/cache/restore@v4
         with:
           path: src/Sentry/Platforms/Native/sentry-native
-          key: sentry-native-macOS-${{ hashFiles('scripts/build-sentry-native.ps1') }}-${{ hashFiles('.git/modules/modules/sentry-native/HEAD') }}
+          key: sentry-native-macos-15-${{ hashFiles('scripts/build-sentry-native.ps1') }}-${{ hashFiles('.git/modules/modules/sentry-native/HEAD') }}
```

Comment:
@jpnurmi We'll prob forget to update the `15`, should we use the agent name as a variable?

Reply:
Good point. It would not be good if it accidentally used the old cache after bumping the version.

This part of the cache key is hard-coded when downloading the cache, because the publishing build has to fetch caches from all runners.